google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/ui/GoogleCloudToolsIcons.java

```diff
@@ -48,13 +46,18 @@
   public static final Icon CLOUD_DEBUG_REACTIVATE_BREAKPOINT = load(
       "/icons/debugreactivatebreakpoint.png");
   public static final Icon CLOUD_DEBUG_DELETE_ALL_BREAKPOINTS = load("/icons/debugdeleteall.png");
+  public static final Icon LOADING = loadGif("/icons/loading.gif");
```

Comment:
does the animation display properly?

Reply:
Yep, as long as we use the `ImageIcon` class directly in `loadGif()`. Using the `IconLoader.getIcon()` method in `.load()` doesn't work.